shot_component:
  ros__parameters:
    port: "/dev/ttyUSB0"
    baudrate: 115200
    pan_servo_id: 11
    trigger_servo_id: 10
    fire_button: 0 # Aボタン（通常0番）
    fire_duration_ms: 300 # 射撃持続時間（ミリ秒）
    pan_axis: 7
    fire_angle: 145.0 # 射撃角度（度）
    home_angle: 245.0 # ホーム角度（度）
    pan_max_angle: 120.0 # パン最大角度（度）
